src/test/kotlin/app/cash/licensee/SpdxLicensesTest.kt

```diff
@@ -15,28 +15,32 @@
  */
 package app.cash.licensee
 
-import org.junit.Assert.assertEquals
-import org.junit.Assert.assertNull
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isNull
 import org.junit.Test
 
 class SpdxLicensesTest {
   @Test fun embeddedDatabaseLitmusTest() {
-    assertEquals(
+    assertThat(
       SpdxLicense("MIT-0", "MIT No Attribution", "https://github.com/aws/mit-0"),
+    ).isEqualTo(
       SpdxLicenses.embedded.findByIdentifier("MIT-0"),
     )
```

Comment:
```suggestion
    assertThat(
       SpdxLicenses.embedded.findByIdentifier("MIT-0"),
    ).isEqualTo(
       SpdxLicense("MIT-0", "MIT No Attribution", "https://github.com/aws/mit-0"),
     )
```
Same for the rest of this file. The expect and the actual are currently reversed for them all.